From c3f484a1f61201ba46b31ff16bc55a8fd5baa1b7 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Tue, 19 May 2020 17:21:42 -0600
Subject: [PATCH 02/14] Add xxxhdpi icon_webview.png to BUILD.gn

---
 android_webview/nonembedded/BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/android_webview/nonembedded/BUILD.gn b/android_webview/nonembedded/BUILD.gn
index d43d8d9b9bda8..936d0bec9fadd 100644
--- a/android_webview/nonembedded/BUILD.gn
+++ b/android_webview/nonembedded/BUILD.gn
@@ -197,6 +197,7 @@ android_resources("icon_resources") {
     "java/res_icon/drawable-mdpi/icon_webview.png",
     "java/res_icon/drawable-xhdpi/icon_webview.png",
     "java/res_icon/drawable-xxhdpi/icon_webview.png",
+    "java/res_icon/drawable-xxxhdpi/icon_webview.png",
   ]
 }
 
-- 
2.53.0

