From ebc7b18509698a1c2896ede5dfd6d14c05371db7 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Thu, 10 Dec 2020 10:09:18 -0500
Subject: [PATCH 06/13] disable WebView variations support

---
 .../android_webview/variations/VariationsSeedLoader.java        | 2 --
 1 file changed, 2 deletions(-)

diff --git a/android_webview/java/src/org/chromium/android_webview/variations/VariationsSeedLoader.java b/android_webview/java/src/org/chromium/android_webview/variations/VariationsSeedLoader.java
index c38b7d343a051..427b9d0763f42 100644
--- a/android_webview/java/src/org/chromium/android_webview/variations/VariationsSeedLoader.java
+++ b/android_webview/java/src/org/chromium/android_webview/variations/VariationsSeedLoader.java
@@ -99,7 +99,6 @@ public class VariationsSeedLoader {
         synchronized (sSeedLoaderLock) {
             if (sSeedLoader == null) {
                 sSeedLoader = new VariationsSeedLoader();
-                sSeedLoader.startVariationsInit();
             }
         }
     }
@@ -116,7 +115,6 @@ public class VariationsSeedLoader {
                     startInit();
                 }
                 assert sSeedLoader != null;
-                sSeedLoader.finishVariationsInit();
                 sSeedLoader = null; // Allow this to be GC'd after its background thread finishes.
             }
         }
-- 
2.53.0

